api/src/trace/span.ts

```diff
@@ -76,6 +77,24 @@ export interface Span {
     startTime?: TimeInput
   ): this;
 
+  /**
+   * Adds a single link to the span.
+   *
+   * Links added after the creation will not affect the sampling decision.
```

Comment:
I would add that adding links at span creation is preferred because of this reason. The spec says that this is a MUST.

Reply:
Added a note about the preference